BitwardenShared/Core/Vault/Repositories/VaultRepositoryTests.swift

```diff
@@ -585,26 +595,40 @@ class VaultRepositoryTests: BitwardenTestCase { // swiftlint:disable:this type_b
         }
     }
 
+    /// `downloadAttachment(_:cipher:)` throws an error if the cipher can't be found in local data storage.
+    func test_downloadAttachment_cipherNotFound() async throws {
+        await assertAsyncThrows {
```

Comment:
🤔 Can you change this to assert which error is thrown? It looks like there's no coverage reported for the error you added so I wonder if an error is being thrown before the database fetch happens?

```
await assertAsyncThrows(error: BitwardenError.dataError("Unable to fetch cipher with ID 2")) { ... }
```

Reply:
I think it was because we were not setting the active account in state service, just fixed this for both test scenarios.